com.chartboost.mediation/Runtime/AdFormats/Banner/ChartboostMediationBannerViewAndroid.cs

```diff
@@ -0,0 +1,161 @@
+#if UNITY_ANDROID
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Chartboost.Banner;
+using Chartboost.Events;
+using Chartboost.Platforms.Android;
+using Chartboost.Requests;
+using Chartboost.Results;
+using Chartboost.Utilities;
+using Newtonsoft.Json;
+using UnityEngine;
+using Logger = Chartboost.Utilities.Logger;
+
+namespace Chartboost.AdFormats.Banner
+{
+    internal class ChartboostMediationBannerViewAndroid : ChartboostMediationBannerViewBase
+    {
+        private readonly AndroidJavaObject _bannerAd;
+        internal Later<ChartboostMediationBannerAdLoadResult> LoadRequest;
+
+        private Dictionary<string, string> _keywords = new Dictionary<string, string>();
+
+        public ChartboostMediationBannerViewAndroid(AndroidJavaObject bannerAd) : base(new IntPtr(bannerAd.HashCode()))
+        {
+            LogTag = "ChartboostMediationBanner (Android)";
+            _bannerAd = bannerAd;
+            
+            // TODO: this can be also done on bridge?   
+            AndroidAdStore.TrackBannerAd(bannerAd);
```

Comment:
What does this mean?

Reply:
Ahhhhh @kushG @cb-jpadilla This refers to the commented code in Kotlin. [UnityBridge.kt](https://github.com/ChartBoost/chartboost-mediation-unity-sdk/pull/166/files#diff-ad0264058fb7cab9992f4cf33d93f74bb0185d494cc33df9de629ea3d2cdcc13)

Reply:
Lol yeah just came here to copy the link but I already see your comment